frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -62,8 +62,8 @@ function FileSearcher:setSearchResults()
         self.results = self.files
     else
         for __,f in pairs(self.files) do
-        DEBUG("f", f)
-            if string.find(string.lower(f.name), string.lower(keywords)) then
+        --DEBUG("f", f)
+            if string.find(string.lower(f.name), string.lower(keywords)) and string.sub(f.name,-4) ~= ".sdr" then
```

Comment:
@Hzj-jie , does the kernel return filename capitalized even if it was created with lower case? I haven't test it myself.

But, since we already have `string.lower(f.name)` calculated in the previous if statement, it's probably worth reusing it for the sdr comparison just on the safer side.

@Frenzie don't comment out the debug line, just remove it. If it's indeed useful, someone can always add it back in the future ;)